whois/requirements.txt

```diff
@@ -1,3 +1,4 @@
-# List third-party dependencies here, separated by newlines. 
+# List third-party dependencies here, separated by newlines.
 # All dependencies must be version-pinned, eg. requests==1.2.0
-# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
\ No newline at end of file
+# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+whois==0.9.7
```

Comment:
Do you need to remove whois being installed from some other location? Was it in the Dockerfile?
JW because IIRC we were using our own fork of whois in the past.

Reply:
Fixed I think.